src/note/commitment.rs

```diff
@@ -78,7 +78,8 @@ impl ExtractedNoteCommitment {
         self.0.to_repr()
     }
 
-    pub(crate) fn inner(&self) -> jubjub::Base {
+    /// Returns the inner [`jubjub::Base`] value
+    pub fn inner(&self) -> jubjub::Base {
```

Comment:
We need this [here](https://github.com/ZcashFoundation/zebra/compare/comm_coords_sapling#diff-49d789823381c8e99c79885d1f62574f0808ad439a348fd11488b1aed14a5a65R243).

Related: The `NoteCommitment` types/impls in `zebra-chain` look unused to me and unnecessary to me, I'm thinking we should remove them if there are no objections.

Reply:
Update: The unused `Notecommitment` types in Zebra were removed at https://github.com/ZcashFoundation/zebra/commit/edabdea52ab9564fac893fdf2932d2aedf39b7dc and https://github.com/ZcashFoundation/zebra/commit/f8a45b89ff71c5cea6d55124942d826dea0d0d96